public/app/features/dashboard-scene/scene/PanelMenuBehavior.tsx

```diff
@@ -102,7 +103,17 @@ export function panelMenuBehavior(menu: VizPanelMenu) {
     });
 
     if (panel.parent instanceof LibraryVizPanel) {
-      // TODO: Implement lib panel unlinking
+      moreSubMenu.push({
+        text: t('panel.header-menu.unlink-library-panel', `Unlink library panel`),
+        onClick: () => {
+          DashboardInteractions.panelMenuItemClicked('unlinkLibraryPanel');
+          dashboard.showModal(
+            new UnlinkLibraryPanelModal({
+              panelRef: (panel.parent as LibraryVizPanel).getRef(),
```

Comment:
Nit: maybe we can set a `const panelParent = panel.parent` at the top level of the function and check if that is `instanceof` and then set it here to avoid betterer issues.

Reply:
👍 fixed